### ISSUE TITLE
Hyphens in less variables names call error in function-calc-no-invalid 

### ISSUE THREAD

**Issue by MrVladevoit** [stylelint/stylelint#4102]
**function-calc-no-invalid / "stylelint": "^9.10.1"**

After adding less variables to calc function i have error in function-calc-no-invalid. It expects me to put space on each side of the "-:

```
div {
  width: calc(100% + @tooltip-padding); // error
}
```
Stylelint prints the errors:

```
Expected space before "-" operator
Expected space after "-" operator
```

**Comment by hudochenkov** [comment]
Please, update your issue [using a template](https://github.com/stylelint/stylelint/issues/new?template=REPORT_A_BUG.md). No need to close and create again with the same text. Edit current issue.

**Comment by ota-meshi** [comment]
@MrVladevoit Hello!
I think you can write as follows.

```less
div {
  width: calc(100% + @{tooltip-padding});
}
```

**Comment by ota-meshi** [comment]
I gave a misinformation.
#4046 has not been released, so changing the syntax may not solve it.

**Comment by hudochenkov** [comment]
Could you try the latest release `stylelint@10.1.0`, please?

**Comment by MrVladevoit** [comment]
@hudochenkov , i am updated stylelint to 10.1.0, but this version have same problem

Code less for example: https://codepen.io/mrvladevoit/pen/xobMJj?editors=0100

Screenshots of this error: 

![image](https://user-images.githubusercontent.com/12032655/59349756-1956c700-8d23-11e9-8fbe-c301a16abecc.png)

![image](https://user-images.githubusercontent.com/12032655/59349760-1bb92100-8d23-11e9-91ff-f54303050774.png)

**Comment by LolliDepp** [comment]
Opened PR #4534 to highlight underlying issues in the test cases

**Comment by jeddy3** [comment]
Closing as duplicate of https://github.com/stylelint/stylelint/issues/4114
